python_ta/transforms/ExprWrapper.py

```diff
@@ -26,13 +27,21 @@ class ExprWrapper:
     node: astroid.NodeNG
     types: Dict[str, str]
 
-    def __init__(self, expr: nodes.Expr, types=None):
-        self.node = expr.value
-        if types is None:
```

Comment:
You should not change the default value of `types`. In general, this was not a good change to make since it was unrelated to your PR.

In this particular case, see https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments.